bundles/io/org.eclipse.smarthome.io.voice/src/main/java/org/eclipse/smarthome/io/voice/internal/extensions/PocketsphinxConsoleCommandExtension.java

```diff
@@ -0,0 +1,302 @@
+import java.util.Collections;
+import java.util.List;
+import java.util.Locale;
+import java.util.Set;
+
+import javax.sound.sampled.AudioFormat;
+import javax.sound.sampled.AudioInputStream;
+import javax.sound.sampled.AudioSystem;
+import javax.sound.sampled.DataLine;
+import javax.sound.sampled.LineUnavailableException;
+import javax.sound.sampled.SourceDataLine;
+import javax.sound.sampled.TargetDataLine;
+
+import org.eclipse.smarthome.io.console.Console;
+import org.eclipse.smarthome.io.console.extensions.AbstractConsoleCommandExtension;
+import org.eclipse.smarthome.io.voice.KSEvent;
+import org.eclipse.smarthome.io.voice.KSListener;
+import org.eclipse.smarthome.io.voice.KeywordSpottingErrorEvent;
+import org.eclipse.smarthome.io.voice.KeywordSpottingEvent;
+import org.eclipse.smarthome.io.voice.RecognitionStopEvent;
+
+/**
+ * Console command extension to interpret human language commands.
+ *
+ * @author Tilman Kamp - Initial contribution and API
+ *
+ */
+public class PocketsphinxConsoleCommandExtension extends AbstractConsoleCommandExtension {
+
+    public PocketsphinxConsoleCommandExtension() {
+        super("ws", "Test Wordspotting.");
+    }
+
+    @Override
+    public List<String> getUsages() {
+        return Collections.singletonList(buildCommandUsage("<command>", "interprets the human language command"));
+    }
+
+    /**
+     * Utility STTListener for testing
+     */
+    class KSListenerUtility implements KSListener {
+        /**
+         * Boolean indicating if session is closed
+         */
+        private boolean isClosed = false;
+
+        /**
+         * Final transcript
+         */
+        private String wordspot;
+
+        /**
+         * Gets if the session is closed
+         *
+         * @return If the session is closed
+         */
+        public boolean isClosed() {
+            return this.isClosed;
+        }
+
+        /**
+         * Gets the transcript
+         *
+         * @return The transcript
+         */
+        public String getSpot() {
+            return this.wordspot;
+        }
+
+        @Override
+        public void ksEventReceived(KSEvent ksEvent) {
+            if (ksEvent instanceof KeywordSpottingEvent) {
+                // this.wordspot = wordspot ksEvent.
+                System.out.println("dentro do listener - found - ");
+
+                return;
+            }
+            if (ksEvent instanceof RecognitionStopEvent) {
+                this.isClosed = true;
+            }
+            if (ksEvent instanceof KeywordSpottingErrorEvent) {
+                KeywordSpottingErrorEvent e = (KeywordSpottingErrorEvent) ksEvent;
+                System.out.println("dentro do listener - found - " + e.getMessage());
+            }
+        }
+    }
+
+    static {
+        NarSystem.loadLibrary();
+    }
+
+    public final native String sayHello();
+
+    @Override
+    public void execute(String[] args, Console console) {
+
+        System.out.println(this.sayHello());
```

Comment:
This is the only code in the `execute` method. This is not all you want this command to do?